raiden/raiden_service.py

```diff
@@ -96,11 +166,12 @@ def __init__(self, chain, private_key_bin, transport, discovery, config):
         transport.protocol = self.protocol
 
         message_handler = RaidenMessageHandler(self)
-        event_handler = RaidenEventHandler(self)
+        state_machine_event_handler = StateMachineEventHandler(self)
+        pyethapp_blockchain_events = PyethappBlockchainEvents()
 
         alarm = AlarmTask(chain)
         # ignore the blocknumber
-        alarm.register_callback(lambda _: event_handler.poll_all_event_listeners())
+        alarm.register_callback(self.poll_blockchain_events)
```

Comment:
Yes, this was always about the blockchain events, made it explicit to differentiate from the Raiden events.